assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.test.js

```diff
@@ -16,6 +16,11 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import { getByText } from '@testing-library/dom';
```

Comment:
This shouldn't be necessary as this query is returned by `render`

Reply:
This is a deliberate choice, as I wanted to use the version of `getByText` which accepts a [container as the first argument](https://testing-library.com/docs/queries/about#using-queries), rather than the [bound version](https://testing-library.com/docs/react-testing-library/api#queries-1) returned by `render`.

Reply:
Thanks! I'm aware of the difference, the part that isn't obvious is why this would be intentionally used/needed instead?